### DISCUSSION TITLE
What is the purpose of prefixing files with "+"?

### DISCUSSION THREAD

**Discussion by bravius** [vikejs/vike#1461] [Discussion] [answered]
Evaluating the vike ssr plugin as an alternative to NextJs. Love that speed boost that vite provides, and it seems pretty straightforward to adopt. 

Just one question though, what is the purpose of the "+" prefix on many of the files in the examples?

I've scanned the documentation but did not find an explanation. If there is one and I missed it, my apologies.

**Comment by bravius** [comment, marked as answer]
I was looking at the routing guide at https://vike.dev/routing, which seemed to have a different file structure. I just looked again and I see it now covers the new design. Also found the explanation in the V1 migration guide. 

Cheers, I am excited about this project and getting away from Next.js

**Reply by brillout** [reply]
(FYI we plan to write a new `vike.dev/tour` guiding the user with his first Vike steps.)
